Firebase/Core/FIRApp.m

```diff
@@ -294,25 +285,13 @@ - (BOOL)configureCore {
   [self checkExpectedBundleID];
   if (![self isAppIDValid]) {
     if (_options.usingOptionsFromDefaultPlist && [self isDataCollectionDefaultEnabled]) {
-      [[NSNotificationCenter defaultCenter]
-          postNotificationName:kFIRAppDiagnosticsNotification
-                        object:nil
-                      userInfo:@{
-                        kFIRAppDiagnosticsConfigurationTypeKey : @(FIRConfigTypeCore),
-                        kFIRAppDiagnosticsErrorKey : [FIRApp errorForInvalidAppID],
-                      }];
+      [FIRCoreDiagnosticsConnector logConfigureCoreWithDefaultPlist];
```

Comment:
We probably don't want to log here if we're not logging errors anymore

Reply:
Removed, so now there's just a single logging call with the minimum data--cool.

Reply:
This change will actually come in with the next PR.